kinogoteborg/src/app/components/profile/Admin.jsx

```diff
@@ -0,0 +1,191 @@
+"use client";
+
+import { useState } from "react";
+
+export default function Admin() {
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [genre, setGenre] = useState("");
+  const [imdbRating, setImdbRating] = useState(5);
+  const [url, setUrl] = useState("");
+  const [alt, setAlt] = useState("");
+  const [releaseDate, setReleaseDate] = useState("");
+  const [movieID, setMovieID] = useState("");
+  const [date, setDate] = useState("");
+  const [error1, setError1] = useState("");
+  const [error2, setError2] = useState("");
+
+  async function onSubmitMovie(e) {
+    e.preventDefault();
+
+    if (!title || !description || !genre || !imdbRating || !url || !alt || !releaseDate) {
+      setError1("Please fill all the fields");
+      return;
+    }
+
+    const genreArray = genre.split(" ");
+
+    try {
+      const res = await fetch("/api/movies", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          title,
+          description,
+          genre: genreArray,
```

Comment:
Might cause problems. 
I did a test move with 3 genre's and there are now 3 copies of the move in the DB